drodrpg/DRODLib/CurrentGame.cpp

```diff
@@ -4936,7 +4939,7 @@ void CCurrentGame::AddRoomsPreviouslyExploredByPlayerToMap(
 
 	for (CIDSet::const_iterator it=this->PreviouslyExploredRooms.begin(); it!=this->PreviouslyExploredRooms.end(); ++it)
 	{
-		AddRoomToMap(*it, bMakeRoomsVisible, false);
+		AddRoomToMap(*it, MapState::Preview, false);
```

Comment:
How is bMakeRoomsVisible used now?

Reply:
It's not used. Since the function is never used without the default arguments, I've removed bMakeRoomsVisible.